src/pages/Result.js

```diff
@@ -55,6 +82,8 @@ export default function Result() {
         name="Go back to overview"
         destination="/"
       ></RedirectButton>
+  </>
```

Comment:
Looks like prettier is not working here. Maybe there is an error in this file?

Reply:
I think I just forgave to save... 😅